07_cmd_lab/cmdlab.txt

```diff
@@ -2,14 +2,26 @@ Intro To Command Line Lab.
 Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
-  - your current shell?
+  - your current shell? 
   - your default shell?
   - a list of all shells currently installed on the machine?
 
+
+-echo $0
+-echo $SHELL
+-cat /etc/shells
+
 עברית שפה קשה. בעקרון המעטפת הנוכחית שלי היא bash
 
 2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
+-Contains all users passwords in the following format:
+login name:optional encrypted password:numerical user ID:numerical group ID:user name or comment field:user home directory:optional user command interpreter
+
```

Comment:
Well it does contain user data, but no password is stored there for a few decades :) You can find the actual encrypted passwords in `/etc/shadow`